_rules/table-header-cell-has-assigned-cells-d0f69e.md

```diff
@@ -27,16 +27,16 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any HTML element with the [semantic role][] of [rowheader][] or [columnheader][] for which all of the following is true:
+This rule applies to any HTML element with is a [semantic][semantic role] [rowheader][] or [columnheader][] for which all of the following is true:
```

Comment:
I don't have edit privileges yet, but "is a" - please remove the "is"

Reply:
```suggestion
This rule applies to any HTML element with a [semantic][semantic role] [rowheader][] or [columnheader][] for which all of the following is true:
```

Is that what you mean, @kherr-a11yb